ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-api-http.j2

```diff
@@ -1,21 +1,25 @@
 {# Management API HTTP #}
-{% if management_api_http is defined and management_api_http is not none %}
+{% if management_api_http is arista.avd.defined %}
 ### Management API HTTP Summary
 
 | HTTP | HTTPS |
 | ---------- | ---------- |
-| {% if management_api_http.enable_http is defined and management_api_http.enable_http is not none %} {{ management_api_http.enable_http | lower }} {% else %} default {% endif %} | {% if management_api_http.enable_https is defined and management_api_http.enable_https is not none %} {{ management_api_http.enable_https | lower }} {% else %} default {% endif %} |
-{%     if management_api_http.enable_vrfs is defined and management_api_http.enable_vrfs is not none %}
+| {% if management_api_http.enable_http is arista.avd.defined %} {{ management_api_http.enable_http | lower }} {% else %} default {% endif %} | {% if management_api_http.enable_https is arista.avd.defined %} {{ management_api_http.enable_https | lower }} {% else %} default {% endif %} |
+{%     if management_api_http.enable_vrfs is arista.avd.defined %}
 
 ### Management API VRF Access
 
 | VRF Name | IPv4 ACL | IPv6 ACL |
 | -------- | -------- | -------- |
 {%         for vrf in management_api_http.enable_vrfs | arista.avd.natural_sort %}
-| {{ vrf }} | {% if management_api_http.enable_vrfs[vrf].access_group is defined and management_api_http.enable_vrfs[vrf].access_group is not none %} {{ management_api_http.enable_vrfs[vrf].access_group }} {% else %} - {% endif %} | {% if management_api_http.enable_vrfs[vrf].ipv6_access_group is defined and management_api_http.enable_vrfs[vrf].ipv6_access_group is not none %} {{ management_api_http.enable_vrfs[vrf].ipv6_access_group }} {% else %} - {% endif %} |
+| {{ vrf }} | {% if management_api_http.enable_vrfs[vrf].access_group is arista.avd.defined %} {{ management_api_http.enable_vrfs[vrf].access_group }} {% else %} - {% endif %} | {% if management_api_http.enable_vrfs[vrf].ipv6_access_group is arista.avd.defined %} {{ management_api_http.enable_vrfs[vrf].ipv6_access_group }} {% else %} - {% endif %} |
 {%         endfor %}
 {%     endif %}
 
+{%  if management_api_http.enable_https is arista.avd.defined and management_api_http.enable_https == true and management_api_http.https_ssl_profile is arista.avd.defined %}
```

Comment:
This block `management_api_http.enable_https is arista.avd.defined and management_api_http.enable_https == true` can be reduced to `management_api_http.enable_https is arista.avd.defined(true)`